git_services/git_services/init/cloner.py

```diff
@@ -153,8 +155,12 @@ def _repo_exists(self):
     def run(self, recover_autosave, session_branch, root_commit_sha):
         logging.info("Checking if the repo already exists.")
         if self._repo_exists():
-            logging.info("The repo already exists - exiting.")
-            return
+            # NOTE: We expect a clean state, if something is left it means the git-clone
```

Comment:
Is that true? What about the case when the pv is left over because something crashed?

Reply:
Oh that is a good point. Yeah I will remove this part.